stack-8.10.3.yaml

```diff
@@ -29,7 +29,7 @@ ghc-options:
   "$everything": -haddock
 
 extra-deps:
-  - brittany-0.13.1.0
```

Comment:
We don't need to add apply-refactor here?

Reply:
yeah, good catch, ci should fail for that stack.yaml